playbooks/roles/deploy-acid/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+# List of additional files to copy to acid/ dir on server
+# Use to e.g. deploy settings.yml, public key for ZuulManager etc.
+files_to_copy: []
+
+flask_env: "development"
+acid_port: 80
+
+# Configuration for database
+
+# dbhost: 127.0.0.1
```

Comment:
Don't do that. Leave only absolute minimum required for Ansible playbook to work.
Less is more. If you prepare something like that but you are not using it you can remove it as well.